multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/Constants.java

```diff
@@ -20,6 +20,8 @@ public class Constants {
     public static final String FILE_ENTRY_DIGEST = "DIGEST";
     public static final String FILE_ENTRY_ID = "FILE_ID";
 
+    public static final String SSL_CLIENT_KEY_FILE = "ClientKey.der";
```

Comment:
Rename it to SSL_CLIENT_KEY_FILE_NAME because this constant provide the name of the file, not the file itself.